decidim-core/app/helpers/decidim/organization_scopes_helper.rb

```diff
@@ -27,7 +27,7 @@ def organization_scopes(organization = current_organization)
     # Arrays, empty Strings, empty Hashes or nil, so if you want to actively filter
     # those elements that do not have a scope, you cannot send an empty String or
     # nil because Searchlight will ignore the parameter. The only solution here is
-    # to use a specific String as ID (`"global"`, in our case), and then treat this
+    # to use a specific Numbre as ID (`0`, in our case), and then treat this
```

Comment:
Typo. This related to what we are discussing in https://github.com/decidim/decidim/pull/1500#discussion_r130640634, isn't it? Is it expected that this changes in `OrganizationScopesHelper`, but the string "global" (or the empty string) is still used in `ParticipatoryProcessScopesHelper`?